Objects/bytearrayobject.c

```diff
@@ -1186,6 +1186,69 @@ bytearray_endswith(PyByteArrayObject *self, PyObject *args)
     return _Py_bytes_endswith(PyByteArray_AS_STRING(self), PyByteArray_GET_SIZE(self), args);
 }
 
+/*[clinic input]
+bytearray.removeprefix as bytearray_removeprefix
+
+    prefix: Py_buffer
+    /
+
+Return a copy of the bytearray with a given prefix removed if present.
```

Comment:
Not sure if it's worth it to mention in the first summary sentence that a copy is returned. It's explained in the whole description below. Again, I like to repeat that prefix a string ;-)

```suggestion
Return a bytearray with a given prefix string removed if present.
```

Same remark for other docstrings.